Source/WebKit/UIProcess/Media/RemoteMediaSessionCoordinatorProxy.cpp

```diff
@@ -59,17 +59,17 @@ RemoteMediaSessionCoordinatorProxy::RemoteMediaSessionCoordinatorProxy(WebPagePr
     : m_webPageProxy(webPageProxy)
     , m_privateCoordinator(WTFMove(privateCoordinator))
 #if !RELEASE_LOG_DISABLED
-    , m_logger(m_webPageProxy->logger())
+    , m_logger(protectedWebPageProxy()->logger())
     , m_logIdentifier(LoggerHelper::uniqueLogIdentifier())
 #endif
 {
     m_privateCoordinator->setClient(*this);
-    m_webPageProxy->legacyMainFrameProcess().addMessageReceiver(Messages::RemoteMediaSessionCoordinatorProxy::messageReceiverName(), m_webPageProxy->webPageIDInMainFrameProcess(), *this);
+    m_webPageProxy->protectedLegacyMainFrameProcess()->addMessageReceiver(Messages::RemoteMediaSessionCoordinatorProxy::messageReceiverName(), protectedWebPageProxy()->webPageIDInMainFrameProcess(), *this);
```

Comment:
Did you consider making webPageIDInMainFrameProcess() inline? If you do, the trivial function detector should kick in, and static analysis will not require a smart pointer for m_webPageProxy. (Same comment in other places in this patch.)

Reply:
Got it. I didn't think that different solution. Thanks.